docs/docs/guides/context.md

```diff
@@ -0,0 +1,131 @@
+# Enrich Apps Traces with Context and Metadata
+
+This guide will walk you through the usage of Context in Keptn.
+
+After reading this guide you will be able to:
+
+- Add metadata to application and workloads traces
+- Link traces between different versions of your application
+
+`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
+and links to traces for a specific application. 
+This enables you to enrich your traces with additional information, 
+making it easier to understand and analyze
+the performance of your applications.
+
+Common usage of this feature may include:
+
+- adding commit id or references related to git-hops tooling
+- referring to different stages and actors
+
+To exploit this feature you need to install Keptn and deploy an application `app.yaml`,
+for instance like done in [step #1-#3 here](../getting-started/observability.md#step-1-install-keptn)
+
+To visualize traces you will require either Jaeger or Grafana:
+
+- Install
+  [Grafana](https://grafana.com/grafana/)
+  or the visualization tool of your choice, following the instructions in
+  [Grafana Setup](https://grafana.com/docs/grafana/latest/setup-grafana/).
+- Install
+  [Jaeger](https://www.jaegertracing.io/)
+  or a similar tool for traces following the instructions in
+  [Jaeger Setup](https://www.jaegertracing.io/docs/1.50/getting-started/).
+
+## Include metadata in workload traces
+
+To enrich the workload traces with custom metadata you can use the
+`keptn.sh/metadata` annotation in your deployment.
+The values specified in the annotation
+are added as key-value attributes to the workload trace.
+
+Modify your `app.yaml` file and change the `app.kubernetes.io/version` from `0.0.2` to `0.0.3`
```

Comment:
Now I get why the app.yaml at the beginning. I think it's a bit confusing the text. Maybe we should add smth like: re-using the prev example <link>, you can modify ...